www/assets/js/components/Facet.tsx

```diff
@@ -31,9 +31,14 @@ function SearchFacet(props: Props) {
       <div
         className="filter-section-title pl-3 pt-2 pb-2"
         onClick={() => setShowFacetList(!showFacetList)}
+        role="button"
+        aria-expanded={showFacetList ? "true" : "false"}
+        tabIndex={0}
```

Comment:
_Here and elsewhere._

**Requested change:** This should be a real button, not a `div` with role `"button"`. 

Currently, the div is focusable but cannot be activated by keyboard events. With a real `button` element, click event handlers are fired when "Enter" or "Space" are pressed. 

Bootstrap's `btn-link` class might help remove some of the unwanted styling.

Reply:
> Ideally, just use buttons now. I suspect it won't actually be that bad. Applying w-100 to a button element gets us most of the styling we want. Maybe bootstrap classes would be helpful; unsure.

I'll try this out.